azure-mgmt-eventhub/src/test/java/com/microsoft/azure/management/eventhub/EventHubTests.java

```diff
@@ -42,7 +42,7 @@ protected void initializeClients(RestClient restClient, String defaultSubscripti
 
     @Override
     protected void cleanUpResources() {
-        if (RG_NAME != null && RG_NAME != "") {
```

Comment:
Fixing incorrect string comparison

Reply:
[P3]: Behavior of `RG_NAME != ""` and `!RG_NAME.isEmpty()` are same. Agree that for readability isEmpty() is better.